Add Process Id to logfiles name (#1076)
github/gh-gei: RELEASENOTES.md

- Allow CLI to fail fast when an unauthorized token is provided by preventing retry logic on 401 errors
- Improve performance when reclaiming a single mannequin with `reclaim-mannequin`
- Improve logging for `reclaim-mannequin` command

> - Log filename now contains the process id to make sure it is unique

Reviewer: ```suggestion
- Add ID of current process to log filenames to ensure that they are unique
```
